browser/src/map/handler/Map.Keyboard.js

```diff
@@ -591,33 +688,33 @@ L.Map.Keyboard = L.Handler.extend({
 		}
 
 		switch (e.keyCode) {
-		case 51: // 3
+		case this.keyCodes.NUM3[0]: // 3
 			if (this._map.getDocType() === 'spreadsheet') {
 				app.socket.sendMessage('uno .uno:SetOptimalColumnWidthDirect');
 				app.socket.sendMessage('commandvalues command=.uno:ViewRowColumnHeaders');
 				return true;
 			}
 			return false;
-		case 53: // 5
+		case this.keyCodes.NUM5: // 5
 			if (this._map.getDocType() === 'spreadsheet') {
 				app.socket.sendMessage('uno .uno:Strikeout');
 				return true;
 			}
 			return false;
-		case 67: // 'C'
-		case 88: // 'X'
-		case 99: // 'c'
-		case 120: // 'x'
-		case 91: // Left Cmd (Safari)
-		case 93: // Right Cmd (Safari)
+		case this.keyCodes.C: // 'C'
+		case this.keyCodes.X: // 'X'
+		case this.keyCodes.NUM3: // 'c'
```

Comment:
@eszkadev I am stuck here, according to the comments, it wants a small 'c' and 'x' but there is no dictionary key-value pairs to replace them, So I am forced to write `NUM3 ` whose corresponding value is 120.
Any leads on what to do?

Reply:
you can create multiple labels with the same value there...
keyCodes.c = 99
keyCodes.NUM3 = 99

but this makes me suspicious, maybe you mixed ASCII codes with some KEY CODES for events here?
Because it should be not ambigous, key codes probably have only one 'C' letter (no small and big variant).
For example if we want big characters - we use the same character code but combined (added) with SHIFT code.

Reply:
As you can see in the code we compare values of `keyCodes` map with `ev.keyCode`.
Documentation is here: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
> Because it should be not ambigous, key codes probably have only one 'C' letter (no small and big variant).

But this raises another question, if There is no small and big variant of 'C' then shouldn't the hardcoded value be `67 `instead of `99`?

Reply:
I think we don't need all the possible values in the map. Only used in the code are enough :)
So from ASCII characters we need only 'c', 'x', 'v', 'z', 'p', 's' if I see correctly :)
Also probabyl we don't need these arrays for NUMX labels...

Reply:
> > Because it should be not ambiguous, key codes probably have only one 'C' letter (no small and big variant).
> 
> But this raises another question, if There is no small and big variant of 'C' then shouldn't the hardcoded value be `67 `instead of `99` in line 706?

documentation says we shouldn't use that field of an event for ASCII characters at all (should be code instead of keyCode - but it's not our problem here)... so... let's keep the code unchanged - we only want human readable names so it's easier to read :)

Reply:
>I think we don't need all the possible values in the map. Only used in the code are enough :)
So should I go with this solution?
keyCodes.c = 99
keyCodes.NUM3 = 99

Reply:
we can unify approach, use arrays for all the cases:
NUM3: [ _, _ ]
C: [ _, _ ]